pkg/master/controller.go

```diff
@@ -263,6 +264,11 @@ func (c *Controller) CreateOrUpdateMasterServiceIfNeeded(serviceName string, ser
 			ClusterIP:       serviceIP.String(),
 			SessionAffinity: api.ServiceAffinityClientIP,
 			Type:            serviceType,
+			SessionAffinityConfig: &api.SessionAffinityConfig{
```

Comment:
> Do we need this for tests to pass?

Yes, currently we should specify` SessionAffinityConfig` when `SessionAffinity` == `api.ServiceAffinityClientIP`